lib/Dialect/TfheRust/IR/TfheRustOps.td

```diff
@@ -30,16 +32,43 @@ class TfheRust_BinaryOp<string mnemonic>
   let results = (outs TfheRust_CiphertextLikeType:$output);
 }
 
+class TfheRust_ScalarBinaryOp<string mnemonic>
+  : TfheRust_Op<mnemonic, [
+    Pure,
+    Commutative,
+    ElementwiseMappable,
+]> {
+  let arguments = (ins
+    TfheRust_ServerKey:$serverKey,
+    TfheRust_CiphertextType:$lhs,
+    AnyTypeOf<[Builtin_Integer, TfheRust_CiphertextType]>:$rhs
```

Comment:
If you want it to be `ElementwiseMappable`, these and the output type should be the `Like` variants that allow tensors. Maybe simpler to remove `ElementwiseMappable` for now

Same below for SubOp